### PR TITLE
container destroy event

### PR DESCRIPTION
**Pull request by tjmehta**

### CODE REVIEW

**Comment by tjmehta** [comment]
@bkendall or @anandkumarpatel

**Comment by anandkumarpatel** [comment]
:+1:

File: test/docker.js [hunk 1]
```diff
@@ -16,7 +17,9 @@ describe('containers', function () {
     async.waterfall([
       docker.createContainer.bind(docker, {}),
       function (container, cb) {
-        container.remove(cb);
+        var count = createCount(cb);
+        eventsStream.on('data', expectStatus('destroy', count.inc().next));
+        container.remove(count.inc().next);
```
Comment (bkendall) [inline test/docker.js]: let's move this test down to the `events` `describe` block. this is testing "should create and delete a container", not that the event was fired (disconnect with the name and what's actually being tested)

Comment (bkendall) [inline test/docker.js]: (I do appreciate the test though :) )

**Comment by bkendall** [comment]
Better. Thank you :)

**Pull request merged**